src/lib.rs

```diff
@@ -37,7 +37,7 @@ where
         .unwrap()
         .write_fmt(format_args!("{}", dot))
         .expect("Writing failed.");
-    let output = child
+    child
```

Comment:
Oups, sorry, This shouldn't have been here. I commented out this part for me, because I didn't want it in `stdout`.

But as said, if you like the general I push a fix.